openmrs-central/prod-openmrs-runtime.properties

```diff
@@ -0,0 +1,5 @@
+connection.url=jdbc:mysql://mysql-central:3306/openmrs_fgh_central?autoReconnect=true&sessionVariables=storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8&maxReconnects=15
+connection.username=root
+auto_update_database=false
+connection.password=root
```

Comment:
Password should no be in the repo, passwords and server certificates should be set by admins on ground since they are province or site specific